src/lib/widgets/user/index.ts

```diff
@@ -0,0 +1 @@
+export { default as LoginContainer } from './ui/login.svelte';
```

Comment:
suggestion (non-blocking): naming components exactly after their filename helps to navigate around the project, I suggest we call this file `login-container` (although, according to my previous comments, I don't think we need this file at all)